doc/source/conf.py

```diff
@@ -0,0 +1,61 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
```

Comment:
The corresponding commit says *"use path from the root of the repository"*, but wouldn't that be: `os.path.abspath(os.path.join('..', '..'))`?

Reply:
How do you invoke `make`? For me it only works if I call it inside `doc`. If I run it from the repository root with (`make -f doc/Makefile`) it complains about not finding `conf.py`.

If I do run it from `doc`, then `os.path.abspath('.')` in `conf.py` returns `/path/to/in-toto/doc/source` and complains about not finding `in_toto`.